sys/net/gnrc/link_layer/lorawan/gnrc_lorawan.c

```diff
@@ -176,7 +176,7 @@ void gnrc_lorawan_event_timeout(gnrc_lorawan_t *mac)
             mac->state = LORAWAN_STATE_RX_2;
             break;
         case LORAWAN_STATE_RX_2:
-            DEBUG("gnrc_lorawan: RX1 timeout.\n");
+            DEBUG("gnrc_lorawan: RX2 timeout.\n");
```

Comment:
Also please don't forget a message for the RX1 timeout :)

Reply:
Already done :+1: